src/agent.c

```diff
@@ -436,6 +437,35 @@ int agent_direct_send(juice_agent_t *agent, const addr_record_t *dst, const char
 	return ret;
 }
 
+int agent_direct_send2(juice_agent_t *agent, const agent_stun_entry_t *entry, const char *data,
```

Comment:
The name `agent_direct_send2` is not very explicit, is there a reason for not modifying `agent_direct_send` since they have the same signature?